grommet/grommet
# Collapsible test

#### What does this PR do?
Adds basic tests to the props for the Collapsible component
#### Where should the reviewer start?
`src/js/components/Collapsible/__tests__/Collapsible-test.js`
#### What testing has been done on this PR?
We have ran `yarn test --coverage` both before and after adding this test file. Coverage breakdown is as follows
Before:
91.67% Statements 44/48
73.33% Branches 22/30
81.82% Functions 9/11
91.49% Lines 43/47
After: 
91.67% Statements 44/48
76.67% Branches 23/30
81.82% Functions 9/11
91.49% Lines 43/47
#### How should this be manually tested?
Run `yarn test --coverage` to make sure the tests run properly and analyze the new coverage
#### Screenshots (if appropriate)
Coverage of collapsible before adding this test file (columns from left to right are, Statements, Branches, Functions, Lines)
<img width="1688" alt="Screen Shot 2020-06-03 at 2 46 20 PM" src="https://user-images.githubusercontent.com/49867133/83683504-80fcc680-a5b3-11ea-9bdd-28e994031dd3.png">
Coverage of collapsible after adding this test file 
<img width="1685" alt="Screen Shot 2020-06-03 at 3 00 01 PM" src="https://user-images.githubusercontent.com/49867133/83683410-5dd21700-a5b3-11ea-91a5-1fb522dfac50.png">

#### Do the grommet docs need to be updated?
No
#### Should this PR be mentioned in the release notes?
No
#### Is this change backwards compatible or is it a breaking change?
Backwards compatible
